spec/api/v1/users_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+describe Api::V1::UsersController do
+  describe 'GET /api/users' do
+    it 'returns all users' do
+      10.times do
+        create(:user)
+      end
+      create(:user, name: 'Janusz', email: 'fake.mail@example.com')
+
+      get '/api/v1/users'
+
+      expect(response.status).to eq(200)
+      expect(JSON.parse(response.body).count).to eq 11
```

Comment:
You can check out if you want a gem called https://github.com/waterlink/rspec-json_expectations but that's completely optional